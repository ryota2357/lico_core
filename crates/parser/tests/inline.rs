//! This file is generated by `cargo codegen parser`, do not edit by hand.
#![cfg_attr(rustfmt, rustfmt::skip)]

mod utils;
const GROUP: &str = "inline";

#[test]
fn expr_literals() {
    let source = include_str!("../test_data/inline/expr_literals.lico");
    let snapshot = utils::make_snapshot("expr_literals", source);
    utils::assert_snapshot!(GROUP, snapshot);
}
#[test]
fn param_list() {
    let source = include_str!("../test_data/inline/param_list.lico");
    let snapshot = utils::make_snapshot("param_list", source);
    utils::assert_snapshot!(GROUP, snapshot);
}
#[test]
fn attr_stmt() {
    let source = include_str!("../test_data/inline/attr_stmt.lico");
    let snapshot = utils::make_snapshot("attr_stmt", source);
    utils::assert_snapshot!(GROUP, snapshot);
}
#[test]
fn break_stmt() {
    let source = include_str!("../test_data/inline/break_stmt.lico");
    let snapshot = utils::make_snapshot("break_stmt", source);
    utils::assert_snapshot!(GROUP, snapshot);
}
#[test]
fn continue_stmt() {
    let source = include_str!("../test_data/inline/continue_stmt.lico");
    let snapshot = utils::make_snapshot("continue_stmt", source);
    utils::assert_snapshot!(GROUP, snapshot);
}
#[test]
fn func() {
    let source = include_str!("../test_data/inline/func.lico");
    let snapshot = utils::make_snapshot("func", source);
    utils::assert_snapshot!(GROUP, snapshot);
}
#[test]
fn return_stmt() {
    let source = include_str!("../test_data/inline/return_stmt.lico");
    let snapshot = utils::make_snapshot("return_stmt", source);
    utils::assert_snapshot!(GROUP, snapshot);
}
#[test]
fn var_stmt() {
    let source = include_str!("../test_data/inline/var_stmt.lico");
    let snapshot = utils::make_snapshot("var_stmt", source);
    utils::assert_snapshot!(GROUP, snapshot);
}
#[test]
fn while_stmt() {
    let source = include_str!("../test_data/inline/while_stmt.lico");
    let snapshot = utils::make_snapshot("while_stmt", source);
    utils::assert_snapshot!(GROUP, snapshot);
}
