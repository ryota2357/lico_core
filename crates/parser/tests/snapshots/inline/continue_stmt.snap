---
source: crates/parser/tests/inline.rs
---
# continue_stmt

## Input

```
continue
```

## CST

```
SOURCE_FILE@0..8
  CONTINUE_STMT@0..8
    CONTINUE_KW@0..8 "continue"
```

## AST

```
SourceFile {
    shebang: none,
    statements: [
        ContinueStmt {
            continue_token: CONTINUE_KW@0..8 "continue",
        },
    ],
}
```

vim:ft=markdown
